src/articles/ArticlePreview.js

```diff
@@ -42,8 +42,11 @@ export default function ArticlePreview({
   }
 
   function titleLink(article) {
+    let linkToRedirect = `/read/article?id=${article.id}`;
+    if (article.last_reading_percentage)
```

Comment:
We could display it for all articles, but currently the way the API call works is that it will only fill it out if it's called with the getUnfinishedUserReadingSessions. We could make make it in generall try to capture this information, but the query to parse the unfinished sessions is not the most efficient - so I would not have it for all articles. We could do it for the most recent top saved articles, or maybe when a user saves an article we could ask the API if there was a previous reading session and then send a little toast - "We have detected you have read up to 50%, do you want to skip to the point you left off" kind of thing? 

I will delete this here, as we don't need it as the unfinished artilcle preview has it's own implementation

Reply:
I guess, we'd have to add this info to the `UserArticle` table, and not compute it every time, but only when we know that a reading session is closed. Compute it then, save it, and then we can send it together with article_info easily.

Reply:
Yes, that could work - but I honestly think it could be a bit much to display it in all articles - maybe if it was in the Saved articles or we had something to see the opened articles it would make a bit more sense. I think storing the value at the end of the session would also be a viable option to save computation time - maybe we revisit this if we see that people use the feature?